### DISCUSSION TITLE
Export a CSV from a Filtered Table

### DISCUSSION THREAD

**Discussion by NathanielCJ99** [sqlpage/SQLPage#308] [Q&A]
I have a table that I have the user filter however they chose. I am wondering if there is an easy way to Export the displayed filtered table as a CSV file? I couldn't find an example of this in the examples or previous discussions.

**Comment by lovasoa** [comment]
Hello! The table component does not offer this possibility. But you can build your own custom csv exports using the csv component: https://sql.ophir.dev/documentation.sql?component=csv#component

**Comment by NathanielCJ99** [comment]
Okay, thanks
